packages/api/endpoints/granules.js

```diff
@@ -242,32 +264,34 @@ const putGranule = async (req, res) => {
   }
 
   try {
-    if (isNewRecord) apiGranule = _setNewGranuleDefaults(apiGranule, isNewRecord);
+    if (isNewRecord) {
+      apiGranule = _setNewGranuleDefaults(apiGranule, isNewRecord);
+    }
     await updateGranuleFromApiMethod(apiGranule, knex, esClient);
   } catch (error) {
     log.error('failed to update granule', error);
     return res.boom.badRequest(errorify(error));
   }
-  return _returnPutGranuleStatus(isNewRecord, apiGranule, res);
+  return _returnPatchGranuleStatus(isNewRecord, apiGranule, res);
 };
 
+function put(req, res) {
+  return res.boom.badRequest('put method not implemented');
+}
+
 /**
- * Applies an 'action' provided with a PUT request
- * e.g. 'move', 'reingest'
+ * Update a single granule by granuleId only.
+ * Supported Actions: reingest, move, applyWorkflow, RemoveFromCMR.
+ * If no action is included on the request, the body is assumed to be an
+ * existing granule to update, and update is called with the input parameters.
  *
  * @param {Object} req - express request object
  * @param {Object} res - express response object
- * @param {Object} pgGranule - Postgres Granule object
- * @param {Object} pgCollection - Postgres Collection object
- * @returns {Promise<Object>} promise of an express response object.
+ * @returns {Promise<Object>} the promise of express response object
  */
-const handlePutAction = async (
-  req,
-  res,
-  pgGranule,
-  pgCollection
-) => {
+async function patchByGranuleId(req, res) {
```

Comment:
This is new. The feature branch had changes to the PUT endpoint to allow either `put` or `putByGranuleId`. The main branch added the PATCH endpoint so those changes needed to be made there. We're allowing users to either PATCH by a granuleId alone or by a granuleId + collectionId.